### PR TITLE
Add short link to community calendar

### PR DESCRIPTION
**Pull request by dims**
Signed-off-by: Davanum Srinivas <davanum@gmail.com>

### CODE REVIEW

File: k8s.io/configmap-nginx.yaml
Comment (nikhita) [inline k8s.io/configmap-nginx.yaml:235]: IIRC this calendar is no longer the "official' one... https://www.k8s.dev/calendar is the new one

cc @mrbobbytables

Comment (mrbobbytables) [reply k8s.io/configmap-nginx.yaml:235]: That is the right calendar, but we do want folks to goto the the k8s.dev site instead. It has more friendly views and doesn't assume PST

Comment (spiffxp) [inline k8s.io/configmap-nginx.yaml:235]:
```suggestion
          rewrite ^/calendar$        https://www.k8s.dev/calendar
```
no objections to adding this with the suggestion

(also I haven't actually deployed changes to this, so I would rather defer to someone else who has for the /approve)

**Review by spiffxp — CHANGES_REQUESTED** [review]
suggestion based on @nikhita and @mrbobbytables reviews

**Comment by spiffxp** [comment]
Thanks to https://github.com/kubernetes/k8s.io/pull/1796 I think I have a playbook to follow here.  Going to followup on https://github.com/kubernetes/k8s.io/pull/1799#pullrequestreview-617963529 first

File: k8s.io/configmap-nginx.yaml
Comment (spiffxp) [reply k8s.io/configmap-nginx.yaml:235]: ping @dims to accept this suggestion

Comment (dims) [reply k8s.io/configmap-nginx.yaml]: good point

**Review by spiffxp — APPROVED** [review]
/approve
/lgtm
/hold cancel

**Comment by spiffxp** [comment]
```
spiffxp@cloudshell:~/k8s.io/k8s.io (kubernetes-public)$ ./deploy.sh canary
# ...
Ran 24 tests in 12.808s

OK
spiffxp@cloudshell:~/k8s.io/k8s.io (kubernetes-public)$ ./deploy.sh prod
# ...
Ran 24 tests in 12.073s

OK
```

**Pull request merged**
